awscrt/websocket.py

```diff
@@ -14,26 +30,25 @@
 from dataclasses import dataclass
 from enum import IntEnum
 import sys
-from typing import Callable, Sequence, Tuple
+from typing import Callable, Optional, Sequence, Tuple, Union
 
 
 def connect(
     *,
```

Comment:
you might really want kwargs here now...

Reply:
Not sure what you mean...
Do you mean that I should take `**kwargs`?

If so, I agree, and I'm doing one better by using the Python 3.0+ feature: [Keyword-Only Arguments](https://peps.python.org/pep-3102/)

Reply:
I missed the * on the first line, my bad.
Thanks for the link!